components/brave_wallet_ui/components/extension/connect-with-site-panel/connect-with-site-panel.tsx

```diff
@@ -69,7 +69,7 @@ import {
 
 const onClickAddAccount = () => {
   chrome.tabs.create(
-    { url: `chrome://wallet${WalletRoutes.AddAccountModal}` },
+    { url: `chrome://wallet${encodeURIComponent(WalletRoutes.AddAccountModal)}` },
```

Comment:
`WalletRoutes.AddAccountModal` is also a hardcoded paths here https://github.com/brave/brave-core/pull/27636#discussion_r1957432756